ark-testing/src/util.rs

```diff
@@ -19,3 +13,34 @@ pub fn init_logging() -> anyhow::Result<()> {
 		.try_init();
 	Ok(())
 }
+
+/// Returns the directory where all test data will be written
+///
+/// By default this is written in the `./test` directory at the project root.
+/// You can also set TEST_DIRECTORY to pick another location.
+/// You are responsible to ensure the `TEST_DIRECTORY` exists
+pub fn test_data_directory() -> PathBuf {
+	match std::env::var_os(TEST_DIRECTORY) {
+		Some(directory) => { PathBuf::from(directory) },
+		None => {
+			let path = get_cargo_workspace().join("test");
+			if ! &path.exists() {
```

Comment:
You don't need this `&` here. All the `Path` methods automatically exist on `PathBuf` because `PathBuf` implements `Deref<Target=Path>`, as explicitly indicated in the docs as well:

![image](https://github.com/user-attachments/assets/c0fb6c7d-4d4a-4578-b9d1-ef8936fe72a3)